### PR TITLE
Chore: Py311 and formatting 

### PR DESCRIPTION
**Pull request by janboll**
Update github mirror to python 3.11 and introduce black and isort formatting tools from qontract-reconcile 

[APPSRE-8449](https://issues.redhat.com/browse/APPSRE-8449)

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+line-length = 88
+target-version = ['py311']
+include = '^.*\.py$'
+
+[tool.isort]
+profile = "black"
+multi_line_output = 3
+force_grid_wrap = 2
+float_to_top = true
+skip_glob = ["reconcile/gql_definitions/*"]
```
Comment (fishi0x01) [inline pyproject.toml:11]: nit: looks like copy/paste left-over

**Review by fishi0x01 — APPROVED** [review]
Nice! :rocket: :heart:

File: pyproject.toml (hunk 1 above)
Comment (janboll) [reply pyproject.toml:11]: fixed

**Review by chassing — APPROVED** [review]
❤️

**Pull request merged**